fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -1082,9 +1089,15 @@ def encode_list(
                     f"Encountered unknown tokens in input starting with {typed_input_list[0].input_string}"
                 )
             elif on_unknown == "warn":
-                warn(
-                    f"Encountered unknown tokens in input starting with {typed_input_list[0].input_string}"
-                )
+                if verbose <= 1:
+                    warning_message = "Encountered unknown tokens in input"
```

Comment:
level 0 should probably be "silent".

Reply:
This wat intentional. I don't want to introduce a completely silent option at this point.